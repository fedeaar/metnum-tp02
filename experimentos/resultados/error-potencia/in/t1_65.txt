-3.149672437859521779e+02 3.842605011052873465e+01 2.893650417462171998e+00 -2.141083835756790776e+01 1.053260235220483736e+02 9.569069687822650394e+01 -2.392941829021427935e+01 -3.137787199135041050e+01 -5.348185574601982495e+01 1.743650313098258948e+02 5.580152158489085679e+01 6.944518840647525337e+00 1.597620936229984352e+02 1.411352462319487131e+02 -3.176892011733276178e+01 2.003570293609634234e+01 3.965076124166273530e+01 8.736852432935788215e+01 -5.754912636093822442e+01 -5.242833518739959686e+01
3.842605011052874175e+01 -2.456994000584445814e+01 -9.082275040322710380e-01 -2.996421827416466499e+01 5.425824579604666553e+01 5.275184789383365569e+01 -7.044990006183843434e+01 -6.114694784458026788e+01 -1.108101750062646573e+02 8.098699525744409300e+01 2.328645444435828793e+01 -7.500260612024374218e-01 7.525597683211194067e+01 7.814967659579390613e+01 -6.895275627526238793e+01 1.198083603968407740e+00 3.878094541051917865e+00 3.366361602095446415e+01 -7.812603780995974034e+01 -8.635790864978856973e+01
2.893650417462171553e+00 -9.082275040322710380e-01 2.289239909581946719e+01 -2.682339936456164420e+00 4.363027273675907480e+00 4.291713319322645326e+00 -6.502594360082483149e+00 -5.565173386097346686e+00 -1.011027574873314627e+01 6.385011858862107204e+00 1.793844236389768643e+00 -1.226708532079377467e-01 5.949985413346063901e+00 6.362646571559323228e+00 -6.302436221393681315e+00 -3.512673873552163428e-02 7.338906301406877830e-02 2.543499154842731702e+00 -6.980893696682990424e+00 -7.799473089961549199e+00
-2.141083835756790776e+01 -2.996421827416466499e+01 -2.682339936456164864e+00 8.890609265293221597e+02 -7.694810457662036640e+00 -3.429832857057008688e+00 -5.812048529983209733e+01 -4.404162334582727567e+01 -8.185300319210925579e+01 -2.184042921612221022e+01 -9.700149043908577795e+00 -4.964802721108168448e+00 -1.892814152752702128e+01 -4.702917913648787085e+00 -5.184584166443352160e+01 -1.086102200969777520e+01 -1.994166793125766191e+01 -1.806754644957717204e+01 -4.573175755716253832e+01 -5.730220768930973918e+01
1.053260235220483594e+02 5.425824579604666553e+01 4.363027273675908368e+00 -7.694810457662038417e+00 -6.197155948753323855e+02 8.879304831041447699e+01 1.617370791683243070e+01 2.745009624774938928e-01 4.883963701159519388e+00 1.719414722769631112e+02 5.678977024526856354e+01 9.512658859270965550e+00 1.568366674458363548e+02 1.307303169231299478e+02 4.999114768089647498e+00 2.519164218455745896e+01 4.884424731927169461e+01 9.078990328923377717e+01 -2.230004540857574469e+01 -1.013869837682300101e+01
9.569069687822651815e+01 5.275184789383364858e+01 4.291713319322645326e+00 -3.429832857057010020e+00 8.879304831041449120e+01 -7.719992779944408312e+02 2.470586042273335892e+01 8.280633899432018552e+00 1.920120184915354145e+01 1.538179990382254516e+02 5.125786646316447559e+01 9.195863136378214620e+00 1.401238890775933612e+02 1.147774090178330653e+02 1.382262361800729700e+01 2.393514110826318131e+01 4.620408784087258880e+01 8.241344424419439463e+01 -1.108228409278828686e+01 1.627758497835827711e+00
-2.392941829021428646e+01 -7.044990006183843434e+01 -6.502594360082483149e+00 -5.812048529983209733e+01 1.617370791683246978e+01 2.470586042273335181e+01 2.990071097723297271e+02 -1.350844253576461824e+02 -2.493238713731165035e+02 1.185820229343693910e+00 -7.241303780331191930e+00 -1.146556131278236279e+01 4.131791276607232888e+00 3.743920116551034027e+01 -1.571660253125544102e+02 -2.334338247770844532e+01 -4.183230522092432579e+01 -1.943474352839001540e+01 -1.492304792307475054e+02 -1.799221769082296589e+02
-3.137787199135042826e+01 -6.114694784458026788e+01 -5.565173386097346686e+00 -4.404162334582727567e+01 2.745009624774867873e-01 8.280633899432020328e+00 -1.350844253576461824e+02 5.084210126355562238e+02 -1.935546977937237045e+02 -2.007295757044541062e+01 -1.253716194479874702e+01 -1.003481666610088219e+01 -1.594898953111538198e+01 1.301558788135696965e+01 -1.222453155556160738e+02 -2.114148012928359321e+01 -3.833807671694063401e+01 -2.612474570615508895e+01 -1.127710883353798295e+02 -1.380088830122633112e+02
-5.348185574601982495e+01 -1.108101750062646573e+02 -1.011027574873314805e+01 -8.185300319210925579e+01 4.883963701159517612e+00 1.920120184915354855e+01 -2.493238713731165603e+02 -1.935546977937237330e+02 2.249600777547253188e+02 -2.965068605990080286e+01 -2.072719612648843679e+01 -1.815849657460303490e+01 -2.267667186910400901e+01 2.979511554589385725e+01 -2.260473884516344185e+02 -3.803182292158037114e+01 -6.882916254960441904e+01 -4.439305272409360015e+01 -2.097026042831800794e+02 -2.558849255539084595e+02
1.743650313098258948e+02 8.098699525744410721e+01 6.385011858862107204e+00 -2.184042921612220312e+01 1.719414722769631112e+02 1.538179990382254516e+02 1.185820229343711674e+00 -2.007295757044542484e+01 -2.965068605990079931e+01 -2.892353043715748413e+02 9.487491612269198527e+01 1.433346960309204476e+01 2.656851587958059895e+02 2.266286428685663168e+02 -1.544533188774700427e+01 3.902547106545038247e+01 7.618796268362635260e+01 1.504820316993777283e+02 -6.037500072304461440e+01 -4.448814413182812189e+01
5.580152158489085679e+01 2.328645444435829148e+01 1.793844236389768865e+00 -9.700149043908579571e+00 5.678977024526857775e+01 5.125786646316447559e+01 -7.241303780331199036e+00 -1.253716194479874346e+01 -2.072719612648843679e+01 9.487491612269198527e+01 -4.493811561022455408e+02 4.165833102110010344e+00 8.682679393226301556e+01 7.556714162979780269e+01 -1.200736603898373467e+01 1.169144036776565976e+01 2.299063677825079210e+01 4.821229265682639920e+01 -2.630759494499790563e+01 -2.248787502224022816e+01
6.944518840647527114e+00 -7.500260612024360896e-01 -1.226708532079377190e-01 -4.964802721108168448e+00 9.512658859270967326e+00 9.195863136378214620e+00 -1.146556131278236634e+01 -1.003481666610088219e+01 -1.815849657460303490e+01 1.433346960309204121e+01 4.165833102110008568e+00 -4.006554109304384781e+01 1.330139043959680478e+01 1.361837085625028720e+01 -1.128744775446340043e+01 3.490973458692281928e-01 9.321203302847971273e-01 6.074861624304386254e+00 -1.295832398999803559e+01 -1.423588422752765936e+01
1.597620936229984920e+02 7.525597683211194067e+01 5.949985413346063901e+00 -1.892814152752701418e+01 1.568366674458363832e+02 1.401238890775933612e+02 4.131791276607243546e+00 -1.594898953111541040e+01 -2.267667186910400545e+01 2.656851587958059326e+02 8.682679393226301556e+01 1.330139043959680478e+01 -3.532851977300254021e+02 2.064339382579166795e+02 -1.132883523056835884e+01 3.607590584757301855e+01 7.036335269978056317e+01 1.378577079087807533e+02 -5.252703914733919532e+01 -3.746539710679381585e+01
1.411352462319486847e+02 7.814967659579392034e+01 6.362646571559323228e+00 -4.702917913648787085e+00 1.307303169231299478e+02 1.147774090178330795e+02 3.743920116551034027e+01 1.301558788135698563e+01 2.979511554589386080e+01 2.266286428685663168e+02 7.556714162979780269e+01 1.361837085625028720e+01 2.064339382579167079e+02 -6.911125476703447248e+02 2.131437332299634946e+01 3.540690889073047742e+01 6.832955910550172973e+01 1.215453977786168878e+02 -1.542844576104091203e+01 3.483704908328903116e+00
-3.176892011733275467e+01 -6.895275627526238793e+01 -6.302436221393681315e+00 -5.184584166443352160e+01 4.999114768089654604e+00 1.382262361800729877e+01 -1.571660253125544102e+02 -1.222453155556160738e+02 -2.260473884516343617e+02 -1.544533188774700250e+01 -1.200736603898373822e+01 -1.128744775446340221e+01 -1.132883523056835706e+01 2.131437332299632104e+01 4.203222545560893764e+02 -2.354028312177175408e+01 -4.254059322335280768e+01 -2.630585160523429167e+01 -1.328753850388687852e+02 -1.618123225669794465e+02
2.003570293609634945e+01 1.198083603968406408e+00 -3.512673873552157877e-02 -1.086102200969777165e+01 2.519164218455745896e+01 2.393514110826317420e+01 -2.334338247770843822e+01 -2.114148012928359321e+01 -3.803182292158037114e+01 3.902547106545036826e+01 1.169144036776566153e+01 3.490973458692276932e-01 3.607590584757302565e+01 3.540690889073047742e+01 -2.354028312177174698e+01 -1.139736231682898904e+02 4.467084055557171141e+00 1.745768797636891634e+01 -2.846120723856726897e+01 -3.053163046174208972e+01
3.965076124166273530e+01 3.878094541051917865e+00 7.338906301406877830e-02 -1.994166793125766191e+01 4.884424731927169461e+01 4.620408784087258880e+01 -4.183230522092432579e+01 -3.833807671694063401e+01 -6.882916254960441904e+01 7.618796268362632418e+01 2.299063677825079210e+01 9.321203302847972383e-01 7.036335269978057738e+01 6.832955910550172973e+01 -4.254059322335282189e+01 4.467084055557171141e+00 -1.283626760439637451e+02 3.451794430899780508e+01 -5.232407227909412484e+01 -5.569727951940780031e+01
8.736852432935786794e+01 3.366361602095446415e+01 2.543499154842731702e+00 -1.806754644957717204e+01 9.078990328923377717e+01 8.241344424419439463e+01 -1.943474352838999408e+01 -2.612474570615510316e+01 -4.439305272409360015e+01 1.504820316993777283e+02 4.821229265682640630e+01 6.074861624304386254e+00 1.378577079087807533e+02 1.215453977786168736e+02 -2.630585160523430943e+01 1.745768797636891634e+01 3.451794430899780508e+01 -3.484565932670835764e+02 -4.861235765495028716e+01 -4.397533911331869660e+01
-5.754912636093822442e+01 -7.812603780995975455e+01 -6.980893696682990424e+00 -4.573175755716254542e+01 -2.230004540857573758e+01 -1.108228409278827975e+01 -1.492304792307475054e+02 -1.127710883353798295e+02 -2.097026042831801078e+02 -6.037500072304460730e+01 -2.630759494499790208e+01 -1.295832398999803559e+01 -5.252703914733918111e+01 -1.542844576104091203e+01 -1.328753850388687852e+02 -2.846120723856726187e+01 -5.232407227909411773e+01 -4.861235765495028716e+01 8.384863876076271936e+02 -1.464536319361476160e+02
-5.242833518739959686e+01 -8.635790864978856973e+01 -7.799473089961550087e+00 -5.730220768930972497e+01 -1.013869837682299391e+01 1.627758497835841922e+00 -1.799221769082296873e+02 -1.380088830122633112e+02 -2.558849255539084879e+02 -4.448814413182813610e+01 -2.248787502224022461e+01 -1.423588422752765936e+01 -3.746539710679378743e+01 3.483704908328903116e+00 -1.618123225669794465e+02 -3.053163046174209327e+01 -5.569727951940780741e+01 -4.397533911331869660e+01 -1.464536319361475876e+02 5.439745281566725907e+02
