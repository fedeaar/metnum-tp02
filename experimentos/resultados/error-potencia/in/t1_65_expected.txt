9.550000000000000000e+02
9.070000000000000000e+02
-8.600000000000000000e+02
-8.500000000000000000e+02
7.250000000000000000e+02
-7.200000000000000000e+02
6.130000000000000000e+02
-5.960000000000000000e+02
5.830000000000000000e+02
-5.800000000000000000e+02
5.630000000000000000e+02
-4.800000000000000000e+02
4.710000000000000000e+02
-4.240000000000000000e+02
-4.160000000000000000e+02
-1.380000000000000000e+02
-1.160000000000000000e+02
-4.000000000000000000e+01
2.300000000000000000e+01
-1.800000000000000000e+01
